.github/workflows/python-release-patch.yml

```diff
@@ -78,3 +78,14 @@ jobs:
           PYPI_USERNAME: ${{ secrets.PYPI_USER }}
           PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
         run: poetry publish -u $PYPI_USERNAME -p $PYPI_PASSWORD -v -n
+
+      - name: Bump repo version
+        run: |
+          git clean -fxd # Clear all the changes
+          poetry version ${{ env.current_version }}.dev0
```

Comment:
For the next build, we don't need to go through all the hoops as we do in line 27-45. We just need to remove `.dev0` from version and run build.

Reply:
Ah, I see what you mean. Yeah, just using the repo's version makes sense!

Reply:
@mederka thanks again for the suggestion, implemented it now!